physionet-django/user/templates/user/application_display_table.html

```diff
@@ -125,10 +125,10 @@
       <td>{% if application.status %}{{ application.get_status_display }}{% else %}<i class="fas fa-clock"></i> Pending{% endif %}</td>
     </tr>
     {% if application.responder_comments %}
-    <tr>
+<!--     <tr>
       <td>Admin Comments</td>
       <td>{{ application.responder_comments|linebreaks }}</td>
```

Comment:
This leaves the comment in the raw HTML. To remove it altogether, the `{{ application.responder_comments|linebreaks }}` should be commented too:

```{{# application.responder_comments|linebreaks #}}```

Reply:
"{{# application.responder_comments|linebreaks #}}" gives a internal server error, and there is no block comment for Django, so I am removing it for now. Because if not then it would be all commented in but seeable in raw HTML or inspect element.

Reply:
my bad, it should just be `{# application.responder_comments|linebreaks #}`

Reply:
Yea, but you will still see the commented HTML in raw or inspected. So I removed it.

Reply:
Okidokes, we can always add later. With `{# application.responder_comments|linebreaks #}` the user should just see `{# application.responder_comments|linebreaks #}` rather than the comment itself, but removing it all for now is good.